clippy_lints_internal/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "clippy_lints_internal"
 version = "0.0.1"
-edition = "2021"
+edition = "2024"
```

Comment:
Is it really part of this PR? Maybe at least a separate commit would be clearer.

Reply:
It was leftover from another change